openreview/arr/arr.py

```diff
@@ -422,9 +422,24 @@ def create_custom_stage(self):
 
     def create_ethics_review_stage(self):
         self.venue.ethics_review_stage = self.ethics_review_stage
-        self.venue.ethics_review_stage.flag_process_path = '../arr/process/ethics_flag_process.py'
-        
-        return self.venue.create_ethics_review_stage()
+        stage_value = self.venue.create_ethics_review_stage()
+        self.client.post_invitation_edit(
+            invitations=self.venue.get_meta_invitation_id(),
+            signatures=[self.venue_id],
+            replacement=False,
+            invitation=openreview.api.Invitation(
```

Comment:
I am reposting this invitation every time because I need to add some ARR specific values, but this invitation has no date process so nothing runs.